### PR TITLE
Populate the net.peer.ip and net.peer.port atttributes

### PR DESCRIPTION
**Pull request by tobiasstadler**

### CODE REVIEW

File: instrumentation/nginx/test/instrumentation/test/instrumentation_test.exs [hunk 1]
```diff
@@ -154,6 +154,8 @@ defmodule InstrumentationTest do
     assert status == 200
 
     assert attrib(span, "net.host.port") == "8000"
+    assert attrib(span, "net.peer.ip") != nil
```
Comment (seemk) [inline instrumentation/nginx/test/instrumentation/test/instrumentation_test.exs:157]:
```suggestion
    assert attrib(span, "net.peer.ip") =~ ~r/\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}/
```

File: instrumentation/nginx/test/instrumentation/test/instrumentation_test.exs [hunk 2]
```diff
@@ -154,6 +154,8 @@ defmodule InstrumentationTest do
...
+    assert attrib(span, "net.peer.port") != nil
```
Comment (seemk) [inline instrumentation/nginx/test/instrumentation/test/instrumentation_test.exs:158]:
```suggestion
    assert attrib(span, "net.peer.port") =~ ~r/\d+/
```

**Review by seemk — APPROVED** [review]
Thanks! Perhaps we should also check for the shape of the attribute? Added my suggestions 🙂

**Comment by tobiasstadler** [comment]
Thank You!

**Pull request merged**